src/lib/Carousel.svelte

```diff
@@ -22,7 +22,8 @@
 {#key carouselImage}
 	<div class="header" style="background-image: url({carouselImage.src});">
 		<div class="details-container shadow">
-			<span class="name">DDD South West</span><br />
+			
+			<h1 class="name">DDD South West Conference</h1><br />
```

Comment:
This change to be an `h1` is great! Unfortunately it seems to be causing the vertical alignment of the text to change and the overall picture box to become larger